Nikita_Korzhavin/hamburger.js

```diff
@@ -0,0 +1,51 @@
+<<<<<<< HEAD
+var Product = require('./api/products.js');
+var CombinedProduct = require('./api/combinedProducts');
+
+function Hamburger(options) {
+  this.SIZE_SMALL = new Product(50, 20, 'small');
+  this.SIZE_LARGE = new Product(100, 40, 'large');
+  this.STUFFING_CHEESE = new Product(10, 20, 'cheese');
+  this.STUFFING_SALAD = new Product(20, 5, 'salad');
+  this.STUFFING_POTATO = new Product(15, 10, 'potato');
+  // this.name для удобного поиска в заказе
+  this.name = options;
+  this.size =
+    options.size === this.SIZE_SMALL.name ? this.SIZE_SMALL : this.SIZE_LARGE;
+  this.stuffing =
```

Comment:
Well, it's quite hard to read already and what it you'll need to add three more stuffings? I'd try to use another way for setting product parameters e.g. using some hashmap
It is also related to other classes like `Drink` and `Salad`

Reply:
fixed